components/browser/tabstray/src/main/java/mozilla/components/browser/tabstray/TabViewHolder.kt

```diff
@@ -23,7 +21,7 @@ class TabViewHolder(
     itemView: View,
     private val tabsTray: BrowserTabsTray
 ) : RecyclerView.ViewHolder(itemView) {
-    private val cardView: CardView = (itemView as CardView).apply {
+    private val view: View = itemView.apply {
```

Comment:
I don't think you need the binding.
```suggestion
    itemView.apply {
```